package.nix

```
@@ -0,0 +1,9 @@
+{ pkgs ? import <nixpkgs> {} , displayrUtils }:
```

```suggestion
{ pkgs ? import <nixpkgs> {}, displayrUtils }:
```

Btw, we did remember the best practice here - found an example:
https://nixos.org/manual/nixpkgs/stable/#auto-generated-8-1.48.3

```
{ pkgs ? import <nixpkgs> {} }:
{
  myTools = pkgs.lib.recurseIntoAttrs {
    inherit (pkgs) hello figlet;
  };
}
```

---

Based on what I read, it just basically let nix recursively evaluates nested attribute set. how does it apply here?